sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionRecord.java

```diff
@@ -98,7 +103,9 @@ private TransactionRecord(
         Map<TokenId, List<TokenNftTransfer>> tokenNftTransfers,
         @Nullable ScheduleId scheduleRef,
         List<AssessedCustomFee> assessedCustomFees,
-        List<TokenAssociation> automaticTokenAssociations
+        List<TokenAssociation> automaticTokenAssociations,
+        @Nullable PublicKey aliasKey,
+        List<TransactionRecord> children
```

Comment:
Is this representing the parent consensus timestamp? I didn't see a children field being returned in the protobuf here: https://github.com/hashgraph/hedera-protobufs/blob/v0.21.0/services/transaction_record.proto

Reply:
understand why children is there now. Please add parent consensus timestamp.